admin_organization.go

```diff
@@ -78,9 +86,7 @@ type AdminOrganizationListOptions struct {
 	// Any organizations with a name or notification email partially matching this value will be returned.
 	Query *string `url:"q,omitempty"`
 
-	// A list of relations to include. See available resources
-	// https://www.terraform.io/docs/cloud/api/admin/organizations.html#available-related-resources
-	Include *string `url:"include"`
+	Include *[]AdminOrgIncludeOps `url:"include"`
```

Comment:
In go, a slice is already passed by reference (really, a pointer and 2 ints) so I don't see a lot of value in making this a pointer type and mostly just not idomatic

Reply:
Slices are actually passed by value (but yeah they hold a pointer to the underlying data we care about). Found this [interesting medium article that discusses using a pointer to a slice](https://medium.com/swlh/golang-tips-why-pointers-to-slices-are-useful-and-how-ignoring-them-can-lead-to-tricky-bugs-cac90f72e77b)

Reply:
OMG! I totally had this conversation about the weirdness of pointer-to-slice practice among some Go libraries with one of my colleagues back at New Relic. But to Brandon's point, I was considering removing this pointer from certain Include fields (though in a different PR) where I am adjusting for instead having ListOptions be a  pointer when passed to a listing functions. So, kind of TBD (if we need pointer for an Include field), depending of which PR gets reviewed first, and then I'll have to make adjustments to the other PR.

Reply:
From the way the I've come to understand the code, list opts represent any optional URL param otherwise it would be an explicit parameter in the method. So ListOptions should always be a pointer, and I think there are some functions that deviate from this consistency. It's rather annoying to have to pass an empty struct when `nil` would suffice nicely. 

As for the point to not point a slice discussion, I would err on using a pointer *but* for our purposes I don't foresee something breaking if we choose the latter. As long as it's consistent, I'm A-OK 👍

Reply:
A different PR #291 addresses the pointer inconsistency on Options. But this is a field within. As long as the empty slice doesnt not translate to an empty URL param (I don't think it does) I think empty slice might be more user friendly